### PR TITLE
Note that Trigger delivers all events if no filter defined

### PR DESCRIPTION
**Pull request by aliok**
# Changes

- :broom: Add rule that Trigger to deliver all events if no filter defined
- See the default `filter` value in https://github.com/knative/specs/blob/main/specs/eventing/spec.md#spec
- Impl: https://github.com/knative/eventing/blob/d8265bb5b08f1bebe61fdecd166f43962b01a0e2/pkg/apis/eventing/v1/trigger_defaults.go#L37
-

/kind enhancement

Fixes #

**Release Note**

```release-note

```

**Docs**

```docs

```

### CODE REVIEW

File: specs/eventing/broker.md [hunk 1]
```diff
@@ -46,6 +46,9 @@ The attributes filter specifying a list of key-value pairs MUST be supported by
 Trigger. Events that pass the attributes filter MUST include context or
 extension attributes that match all key-value pairs exactly.
 
+If the attributes filter is not provided, Trigger MUST deliver all events to
```
Comment (aliok) [inline specs/eventing/broker.md:49]: Not sure if it is a `MUST` or a `SHOULD` or something else.

Comment (matzew) [reply specs/eventing/broker.md:49]: I think it's a MUST.

if not all get delivered, imo it's broken :-)

**Pull request merged**